### PR TITLE
Add Italian translations [Resolves #213]

### LINKED ISSUE #213: Site Translation: Italian
https://crowdin.com/project/ethereumfoundation/it#

### PR DESCRIPTION
**Pull request by samajammin**
## Related Issue
https://github.com/ethereum/ethereum-org-website/issues/213

## How Has This Been Tested?
Locally

## Checklist:

- [x] I have read the contributing guidelines in the project README.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/it/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Casa
```
Comment (hacktar) [inline docs/it/index.md]: "Home" cannot be translated in "Casa"... please leave "home" untranslated. 
Home is commonly used even in italian.

File: docs/it/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+meta:
+  - property: og:title
+    content: Casa | Ethereum
```
Comment (hacktar) [inline docs/it/index.md]: Home | Ethereum, like above

File: docs/.vuepress/theme/utils/translations.js [hunk 3]
```diff
@@ -94,6 +94,52 @@ const translations = {
     'page-learn': "Apprendre",
     'page-developers': "Développeurs"
   },
+  'it-IT': {
+    path: '/it/',
+    language: 'Italiano',
+    ethereum: 'Ethereum',
+    'link-text-artwork': 'Leggi le nuove opere d\'arte!',
```
Comment (hacktar) [inline docs/.vuepress/theme/utils/translations.js]: Approfondisci sull'elaborato artistico!

**Comment by samajammin** [comment]
Thanks for the review! Made those updates.

**Comment by samajammin** [comment]
@hacktar we're just waiting on a final review on this from our translations team. We should have this shipped by Devcon next week. 🚀

**Pull request merged**